examples/roitberg/pyanitools.py

```diff
@@ -0,0 +1,126 @@
+# Written by Roman Zubatyuk and Justin S. Smith
+import h5py
+import numpy as np
+import platform
+import os
+
+PY_VERSION = int(platform.python_version().split('.')[0]) > 3
+
+
+class datapacker(object):
```

Comment:
In that case, I think we should modify to `Dataloader`. Lower case class names are bad for maintenance (even if `examples/` isn't formally part of DeepChem, it's where new users start)